packages/core/strapi/lib/types/core/config/shared.d.ts

```diff
@@ -0,0 +1,7 @@
+export type ConfigFunction<TConfigObject = UnknownConfigObject> = ({ env }) => TConfigObject;
```

Comment:
Isn't this throwing an error due to implicit any type on the params.env?

Reply:
Shouldn't we type the function params?